modules/aws-codebuild-step/main.tf

```
@@ -0,0 +1,108 @@
+/**
+ * # AWS codebuild step
+ * AWS codebuild step is very simple module for creating CodeBuild project using just one object in terraform.
+ * This module creates roles and codebuild project.
+ */
+
+data "aws_caller_identity" "current" {}
+data "aws_region" "current" {}
+
+resource "aws_iam_role" "role" {
+  name = "${var.project_name}-role"
+
+  assume_role_policy = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Principal": {
+        "Service": "codebuild.amazonaws.com"
+      },
+      "Action": "sts:AssumeRole"
+    }
+  ]
+}
+EOF
+}
+
+data "aws_iam_policy_document" "task_role_assume" {
```

It's not assume role policy, I'd change that to just:
```suggestion
data "aws_iam_policy_document" "task" {
```